test/language/module-code/source-phase-import/import-source-binding-name_FIXTURE.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2024 Chengzhong Wu. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+// ImportBinding in ImportDeclaration may be 'source' and 'from'
+// This test is verified with `import-source.js` in the same directory that
+// this file does not raise SyntaxError. Note that a SyntaxError could also
+// be raised when the imported module does not have a source phase
+// representation (see sec-source-text-module-record-initialize-environment, 7.c.ii).
+//
+// esid: sec-modules
+// info:
+//   ImportDeclaration:
+//     import source ImportedBinding FromClause ;
+
+$DONOTEVALUATE();
```

Comment:
We should remove this. If it ever gets called, it'll throw an error, trigger the `.catch` block in `import-source.js` with an error that's not a SyntaxError, and make the test pass.